src/nvmeof/NVMeofGwMonitorClient.h

```diff
@@ -96,6 +100,8 @@ class NVMeofGwMonitorClient: public Dispatcher,
   void disconnect_panic();
 
   void handle_nvmeof_gw_map(ceph::ref_t<MNVMeofGwMap> m);
+
+  void connect_panic();
```

Comment:
suggestion: check_connect_panic()

Reply:
@athanatos thank you for review. I see where you’re coming from. Since we already have disconnect_panic() at line 100, I’ll keep it as is for consistency.